packages/babel-plugin/src/shared/messages.js

```diff
@@ -12,63 +12,42 @@
 // This file contains constants to be used within Error messages.
 // The URLs within will eventually be replaced by links to the documentation website for Stylex.
 
-export const ILLEGAL_ARGUMENT_LENGTH =
-  'stylex.create() should have 1 argument.';
-export const ILLEGAL_ARG_LENGTH_FOR_KEYFRAMES =
-  'stylex.keyframes() should have 1 argument.';
-export const NON_STATIC_VALUE =
-  'Only static values are allowed inside of a stylex.create() call.';
-export const NON_STATIC_KEYFRAME_VALUE =
-  'Only static values are allowed inside of a stylex.keyframes() call.';
+export const illegalArgumentLength = (fn: string, argLength: number): string =>
+  `${fn}() should have ${argLength} argument${argLength === 1 ? '' : 's'}.`;
+export const nonStaticValue = (fn: string): string =>
+  `Only static values are allowed inside of a ${fn}() call.`;
+export const nonExportNamedDeclaration = (fn: string): string =>
+  `The return value of ${fn}() must be bound to a named export.`;
+export const unboundCallValue = (fn: string): string =>
+  `${fn}() calls must be bound to a bare variable.`;
+export const nonStyleObject = (fn: string): string =>
+  `${fn}() can only accept a style object.`;
```

Comment:
Before, we'd say "style object" for create calls but just "object" for keyframes. Looking at how this function is now used, I think "can only accept an object" is the message we want.